shared/fs/banner/conflict-banner-container.tsx

```diff
@@ -12,25 +13,26 @@ type OwnProps = {
 const mapStateToProps = (state, ownProps: OwnProps) => ({
   _tlf: Constants.getTlfFromPath(state.fs.tlfs, ownProps.path),
 })
+
 type DispatchProps = {
   onFeedback: () => void
   onFinishResolving: () => void
+  onGoToTlf: (tlfPath: Types.Path) => void
   onHelp: () => void
-  onSeeOtherView: () => void
   onStartResolving: () => void
 }
 const mapDispatchToProps = (dispatch, ownProps: OwnProps) => ({
   onFeedback: () => {},
-  onFinishResolving: () => {},
+  onFinishResolving: dispatch(FsGen.createFinishManualConflictResolution({localViewTlfPath: ownProps.path})),
+  onGoToTlf: (tlfPath: Types.Path) =>
+    dispatch(RouteTreeGen.createNavigateAppend({path: [{props: {path: tlfPath}, selected: 'main'}]})),
```

Comment:
It could be cool to route to the corresponding subfolder when you click on the banner in a subfolder of the conflict TLF. Not necessarily a thing for this particular PR, but a cool thing to have in the future if you agree

Reply:
AH that's right. Will change!